packages/projection/test/operators/Mappers/handleUtil.ts

```diff
@@ -75,6 +77,14 @@ export const referenceTokenAssetName = Asset.AssetNameLabel.encode(
   handleAssetName(maryHandleOne),
   Asset.AssetNameLabelNum.ReferenceNFT
 );
+export const virtualHandleAssetName = Asset.AssetNameLabel.encode(
+  handleAssetName(virtualHandle),
+  Asset.AssetNameLabelNum.UserNFT
```

Comment:
According to https://public.koralabs.io/documentation/HandleResolution.pdf the label for virtual subhandles is 0
```suggestion
  0
```

Reply:
addressed: 101ffed886a656361186adf3e31b20f62abd7089